src/eth/rpc/rpc_server.rs

```diff
@@ -380,7 +360,7 @@ async fn stratus_get_subscriptions(_: Params<'_>, ctx: Arc<RpcContext>, ext: Ext
 // -----------------------------------------------------------------------------
 
 async fn net_listening(params: Params<'_>, arc: Arc<RpcContext>, ext: Extensions) -> Result<JsonValue, RpcError> {
-    stratus_readiness(params, arc, ext).await
+    stratus_health(params, arc, ext).await
```

Comment:
If you are going to put this check inside `net_listening`, we will need some tracing here because connected clients can also use this operation, not only the health check.